test/integration/runtime-basic/basicDom.json

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
in my PR i gave the file a more explicit name like `twoTextFields`, maybe it helps understand better the contents of the DOM without having to look at the JSON content itself.